<launch>
  <arg name="depth_topic" default="/camera/aligned_depth_to_color/image_raw" />
  <arg name="rgb_topic" default="/camera/color/image_raw/compressed" />
  <arg name="camera_info_topic" default="/camera/aligned_depth_to_color/camera_info" />
  <arg name="stretch_robot_rotate_image_90deg" default="true" />

  <node
    pkg="feature_extractor"
    name="right_wrist_node"
    launch-prefix="$(find feature_extractor)/.venv/bin/python"
    type="python_entrypoint.py"
    output="screen"
  >
    <param name="depth_topic" value="$(arg depth_topic)" />
    <param name="rgb_topic" value="$(arg rgb_topic)" />
    <param name="camera_info_topic" value="$(arg camera_info_topic)" />
    <param name="stretch_robot_rotate_image_90deg" value="$(arg stretch_robot_rotate_image_90deg)" />
  </node>

</launch>
